### DISCUSSION TITLE
Validation function feature request

### DISCUSSION THREAD

**Discussion by Sam115200** [Shopify/function-examples#485] [Feedback: Cart and Checkout Validation API]
Would it be possible to show a message to customer using the validation function, but **not block the checkout**?

There are many merchants who want to show a message to customers under certain situations, but do not want to block the checkout, just a simple notification to relevant customers.

Perhaps this can be added in validation function output, where we pass a parameter which says checkout blocking is false?

**Comment by gil--** [comment]
Isn't this the use-case for a UI Extension?

**Reply by Sam115200** [reply]
Yes, but it would be great if functions can also support it, as it seems trivial to add this, but its a good feature.
